### ISSUE TITLE
Question: Adding type information to exporter input parameters?

### ISSUE THREAD

**Issue by rimi-itk** [pretix/pretix#1849]
I'm testing https://docs.pretix.eu/en/latest/api/resources/exporters.html and it works excellently!

One small thing I'm missing is information on the type of an exporter input parameter.

In the [example response](https://docs.pretix.eu/en/latest/api/resources/exporters.html#get--api-v1-organizers-(organizer)-events-(event)-exporters-)

```
    "input_parameters": [
        {
          "name": "_format",
          "required": true,
          "choices": [
            …
          ]
        },
        {
          "name": "paid_only",
          "required": false
        }
      ]
```

we can guess that `_format` has the type `choice`, say, but we don't know anything about `paid_only` (apart from the name which hints at it being a boolean).

Will it make sense to add type information to the input parameter information? I've played around with doing so in https://github.com/pretix/pretix/compare/master...rimi-itk:feature/exporter-input-parameter-type-name, but I'm not sure I'm doing it correctly – or if it makes sense to do it in this way.

Best regards,
Mikkel

**Comment by raphaelm** [comment]
Yeah, the main reason I did not add a type was because I did not know how to do this automatically in a useful way. The approach in your branch is probably a little too naive, since Django field classes do not always represent a type. For example:

* ``SplitDateTimeField`` shouldn't be ``split_date_time``, just ``datetime``
* ``ModelMultipleChoiceField`` should probably just be ``choice``
* ``MyCustomInputField`` should probably just be ``string``

Also, these still aren't really JSON-level types, which would be ``string`` in almost all cases, although types like ``datetime`` are helpful to know even though they don't exist in JSON. I think the only way would be to have a manually-created mapping from Django field classes to the types we wanna show with a fallback to "unknown" or something like that.
